modules/timepoint_list/templates/menu_timepoint_list.tpl

```diff
@@ -6,6 +6,9 @@
     <th>
       DOB
     </th>
+    <th>
+      Age
```

Comment:
Maybe this should be 'Age in Months' since that's what you're returning above?

Reply:
i only added this to show an implementation of the new feature. not sure if we want it to be merged. i can add "(months)" but what if people actually want in years?  or not show the age at all?

Reply:
Maybe a config option should be added then so that projects can decide?

Reply:
then this PR becomes a whole more than just introducing these new functions. maybe a separate PR for implementation can be opened

Reply:
Works for me. I think this part of the code should be made consistent or deleted before merging though

Reply:
yepp, i mentioned deleting before merging in the PR description